packages/eslint-plugin-next/src/rules/no-before-interactive-script-outside-document.ts

```diff
@@ -1,9 +1,10 @@
-const path = require('path')
+import { defineRule } from '../utils/define-rule'
+import * as path from 'path'
```

Comment:
The `import * as from` syntax is used as the `esModuleInterop` is not enabled in the `.swcrc` and `tsconfig.json`.